aggregator/aggregator.go

```diff
@@ -307,10 +318,18 @@ func (agg *Aggregator) Close() error {
 func (agg *Aggregator) sendAggregatedResponseToContract(blsAggServiceResp blsagg.BlsAggregationServiceResponse) {
 	// TODO: check if blsAggServiceResp contains an err
```

Comment:
Todo can be removed

Reply:
Done in [b199d0f](https://github.com/NethermindEth/near-sffl/pull/159/commits/b199d0fc0ef92ecefdfa639b23d8a228401faeac)